lib/date.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+
+const _ = require('lodash');
+const path = require('path');
+const FormController = require('hof-form-wizard').Controller;
```

Comment:
I would probably not use this controller since the difference between the hof-form-wizard controller and hof-form-controller is session io, and I would not expect field-level components to be performing session io directly.

I'd look to write a separate "field component" controller which locked down what actions a field component could perform a lot tighter.